eZ/Bundle/EzPublishRestBundle/Resources/config/default_settings.yml

```diff
@@ -4,3 +4,83 @@ parameters:
 
     # URI part that all REST routes begin with. By this a REST request is recognized.
     ezpublish_rest.path_prefix: /api/ezp/v2
+
+    ezpublish_rest.root_resources:
```

Comment:
I like this, this simpler.  In `service.yml` I can just pass `'$rest_root_resources$'`.  We would lose the checks from the semantic config.